### PR TITLE
Vic 201 - Modal window

### PR DESCRIPTION
**Pull request by mattstryfe**
[Detailed Notes
](https://radiantsolutions.atlassian.net/browse/VIC-201)
- Upgrades
- Angular 1.4.8 → 1.7.5
- Angular Router 0.0.15 → 1.0.20
- Angular UI Bootstrap 0.14.3 → 2.5.0
- Angular Sanitize 1.4.8 → 1.7.8
- Font Awesome 4.4.0 → 5.9.0

To test...
- Check [Videos Only]
- Press [Modal Button] on tile list
- Does it display the video's metadata?
- Does the toolbox tab display correctly
- Does the Download button function properly?
- Does the Share button function properly?

Other tests...
With the upgrades to angular and font awesome, there may be some other quirky things happening throughout the app.  I did a ton of testing but I'm positive I did not check everything.  please poke around and pay close attention to missing icons and other various ui related things.

### CODE REVIEW

File: apps/omar-ui-app/grails-app/conf/application.yml [hunk 1]
```diff
@@ -73,7 +73,7 @@ omar:
       # Message of The Display Banner settings
       motd:
         # Used to show/hide the message of the day banner
-        enabled: true
+        enabled: false
```
Comment (knaquin) [inline apps/omar-ui-app/grails-app/conf/application.yml]: What's the reason for the change here?

File: apps/omar-ui-app/grails-app/conf/application.yml [hunk 2]
```diff
@@ -92,7 +92,7 @@ omar:
       splashModal:
         delay: 1000
         # Used to enable/disable the splash modal from displaying
-        enabled: true
+        enabled: false
```
Comment (knaquin) [inline apps/omar-ui-app/grails-app/conf/application.yml]: What's the reason for the change here?

File: build.gradle [hunk 3]
```diff
@@ -2,6 +2,7 @@ buildscript {
   repositories {
     mavenLocal()
     maven { url ossimMavenProxy }
+    mavenCentral()
```
Comment (knaquin) [inline build.gradle]: Let's remove this one, if there are components you need from mavenCentral, we should modify our nexus so that they can be satisfied.

File: settings.gradle [hunk 4]
```diff
@@ -1,6 +1,7 @@
 pluginManagement {
   repositories {
     maven { url ossimMavenProxy }
+    mavenCentral()
```
Comment (knaquin) [inline settings.gradle]: The only one you should need is the ossimMavenProxy. There is no mavenCentral on JWICS.

File: apps/omar-ui-app/grails-app/conf/application.yml (hunk 1 above)
Comment (mattstryfe) [reply apps/omar-ui-app/grails-app/conf/application.yml]: Forgot to flip that back for local dev.

File: apps/omar-ui-app/grails-app/conf/application.yml (hunk 2 above)
Comment (mattstryfe) [reply apps/omar-ui-app/grails-app/conf/application.yml]: Forgot to flip that back for local dev.

File: build.gradle (hunk 3 above)
Comment (mattstryfe) [reply build.gradle]: Rgr.  Added that when i began upgrading things.  Seems to compile fine without.

File: settings.gradle (hunk 4 above)
Comment (mattstryfe) [reply settings.gradle]: Rgr.  Added that when i began upgrading things.  Seems to compile fine without.
